01-backup.md

```diff
@@ -803,6 +803,31 @@ $ git status --ignored
 nothing to commit, working directory clean
 ~~~
 
+> ## Committing Changes to Git {.challenge}
+> Which command(s) below would save changes of `myfile.txt` to my local git repository?
```

Comment:
We capitalize "Git" unless referring specifically to the command line program. (Also on line 821.)